### PR TITLE
Fix minor typo in setting.js

### PR DESCRIPTION
**Pull request by ashwathbhat**
This PR fixes minor typo in setting.js.

I request the maintainers to review this PR.

### CODE REVIEW

File: src/color/setting.js [hunk 1]
```diff
@@ -462,7 +462,7 @@ p5.prototype.colorMode = function(mode, max1, max2, max3, maxA) {
  * 60x60 light green rect with black outline in center of canvas.
  * 60x60 soft green rect with black outline in center of canvas.
  * 60x60 red rect with black outline in center of canvas.
- * 60x60 dark fushcia rect with black outline in center of canvas.
+ * 60x60 dark fushsia rect with black outline in center of canvas.
```
Comment (stalgiag) [inline src/color/setting.js]: I think the correct spelling might be [fuchshia](https://en.wikipedia.org/wiki/Fuchsia_(color)). That's a tough-ie!

File: src/color/setting.js [hunk 2]
```diff
@@ -304,7 +304,7 @@ p5.prototype.clear = function() {
  *                              current color mode
  * @param {Number} max2     range for the green or saturation depending
  *                              on the current color mode
- * @param {Number} max3     range for the blue or brightness/lighntess
+ * @param {Number} max3     range for the blue or brightness/lightess
```
Comment (stalgiag) [inline src/color/setting.js]: Dropped the 'n'. I think 'lightness' is correct here.

File: src/color/setting.js [hunk 3]
```diff
@@ -714,7 +714,7 @@ p5.prototype.noStroke = function() {
  * 60x60 white rect at center. Bright green outline.
  * 60x60 white rect at center. Soft green outline.
  * 60x60 white rect at center. Red outline.
- * 60x60 white rect at center. Dark fushcia outline.
+ * 60x60 white rect at center. Dark fushsia outline.
```
Comment (stalgiag) [inline src/color/setting.js]: See above

File: src/color/setting.js (hunk 1 above)
Comment (ashwathbhat) [reply src/color/setting.js]: Oh yeah.
Minor error from my side. Will fix it.
Thank you @stalgiag

Comment (ashwathbhat) [reply src/color/setting.js]: @stalgiag I think it's fuchsia and not fuchshia.

Comment (stalgiag) [reply src/color/setting.js]: Haha yes. Typos on typos.

**Comment by stalgiag** [comment]
Great thanks!

**Pull request merged**